openfisca_web_api_preview/loader/variables.py

```diff
@@ -17,8 +17,10 @@ def get_default_value(variable):
     default_value = variable.default_value
     if isinstance(default_value, datetime.date):
         return default_value.isoformat()
+
     if variable.value_type == Enum:
```

Comment:
Use `default_value` (line 18)?
Set the condition on `default_value` instead of `variable.value_type` to make it similar to the datetime condition?